### PR TITLE
[LW-10261] add posthog cancel dapp tx events 

### PR DESCRIPTION
**Pull request by VanessaPC**
# Checklist

- [x] JIRA -LW-10261
- [ ] Proper tests implemented
- [x] Screenshots added.

---

## Proposed solution

Add posthog actions on both confirm and cancel external transactions. 

## Testing
For instance: 
Go to: https://app.preprod.wingriders.com/welcome
Try to create a transaction, and confirm it. 
Do it again and cancel it, you should see `send| confirm transaction | cancel | click`

### CODE REVIEW

File: packages/common/src/analytics/types.ts [hunk 1]
```diff
@@ -136,6 +136,7 @@ export enum PostHogAction {
   SendTransactionDataReviewTransactionClick = 'send | transaction data | review transaction | click',
   SendTransactionSummaryConfirmClick = 'send | transaction summary | confirm | click',
   SendTransactionConfirmationConfirmClick = 'send | transaction confirmation | confirm | click',
+  SendTransactionConfirmationCancelClick = 'send | transaction summary | cancel | click',
```
Comment (lucas-barros) [inline packages/common/src/analytics/types.ts:139]:
```suggestion
  SendTransactionConfirmationCancelClick = 'send | transaction confirmation | cancel | click',
```

Comment (tomislavhoracek) [reply packages/common/src/analytics/types.ts:139]: 👍

**Review by tomislavhoracek — COMMENTED** [review]
I'm ready to approve it once the change request Lukas suggested is applied

File: packages/common/src/analytics/types.ts (hunk 1 above)
Comment (VanessaPC) [reply packages/common/src/analytics/types.ts]: addressed here: 6d415a4f3da5e2897b9c55f7a9fc3048095e948f

**Review by tomislavhoracek — APPROVED** [review]

**Review by tomislavhoracek — APPROVED** [review]

**Review by lucas-barros — APPROVED** [review]

**Review by DominikGuzei — APPROVED** [review]

**Review by bslabiak — APPROVED** [review]

**Pull request merged**